network/src/peers/peers.rs

```diff
@@ -380,7 +380,7 @@ impl Node {
         let max_peers = self.config.maximum_number_of_connected_peers() as usize;
 
         if num_connected > max_peers {
-            warn!(
+            info!(
```

Comment:
```suggestion
            debug!(
```
this change makes the log less "scary", but this information is probably a bit too verbose for regular users